### PR TITLE
008 merge conflicts resolved

### PR DESCRIPTION
**Pull request by TheoHodges**

### CODE REVIEW

File: tests/training.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            return Promise.resolve(expected);
+        });
+
+        let results = await getValidBands();
+
+        expect(results).toBe(expected.data);
+
+        expect(mockAxios.get).toHaveBeenCalledWith('http://localhost:8080/band/');
+    })
+})
```
Comment (CKindred) [inline tests/training.test.js:49]: Could maybe add a rejection test here?

**Review by sebastianczajkowski — CHANGES_REQUESTED** [review]
Rejection tests

**Review by CKindred — APPROVED** [review]
Good stuff!

**Review by sebastianczajkowski — APPROVED** [review]
Looks good

**Pull request merged**